### ISSUE TITLE
Pumps

### ISSUE THREAD

**Issue by spifftek70** [akulo/pentair-pool-controller#1]
More of a question than an issue.  You have one Pentair IntelliCom II Interface Adapter for RS485 coms to the Intelliflo pump.  But then your Relay board lists multiple pumps...  The input is the yellow and green cable from one pump, but I'm not understanding how that leads to multiple pumps for control.   Also, can a Node-Red board be used in place of the IntelliCom II Interface Adapter?  Do you need one adapter per pump?

**Comment by coreyfournier** [comment]
I think he is using the super flow vs that supports rs485 and 4 5v triggers for each speed. That's what I'm using.

**Comment by akulo** [comment]
@spifftek70 Its not multiple pumps. Its just 4 different programs/speeds for same pump. I am using one Pentair IntelliCom II Interface Adapter and Intelliflo pump.

**Comment by jaysalazar** [comment]
@akulo , Follow up question regarding the intelliCom II Interface Adapter. How exactly are you setting the RPM settings for each program/speed? Does it use the preset speeds on the pump?

**Comment by spifftek70** [comment]
When setting up the schedule for your pool, it allows you to set the pump speed. You can also dynamically change the speed of the pump in the regular dashboard interface.

**Comment by jaysalazar** [comment]
I think I am missing something, looking at the node red flow, I don't understand how you are able to send a command to change the speed of the pump. Is that what you meant by "regular dashboard interface"?

**Comment by akulo** [comment]
yes on pump itself I preprogrammed 4 different speeds. All I do from node-red flow just trigger one of those speeds. I cant dynamically change speeds from my controller. All I can do is select one of the 4 pre-programmed speeds. Read your pump manual only certain programs allow such control as far as I remember.

**Comment by jaysalazar** [comment]
Thanks for confirming, planning on using relays to trigger each speed. I found an example online, my wiring a slightly different but the concept is the same. 

![image](https://github.com/akulo/pentair-pool-controller/assets/63948569/9b66ae21-0a68-40f8-a5c8-c595b5a4750a)
